src/coreclr/tests/testgrouping.proj

```diff
@@ -10,7 +10,7 @@
       <XUnitWrapperDll>$(BinDir)JIT\jit64\JIT.jit64.XUnitWrapper.dll</XUnitWrapperDll>
     </TestGrouping>
 
-    <TestGrouping Condition="'$(TargetOS)' == 'Windows_NT'" Include="$(BinDir)JIT\jit64\mcc\**">
+    <TestGrouping Condition="'$(TargetOS)' == 'Windows_NT'" Include="$(BinDir)JIT\jit64\mcc\**"  Exclude=”$(BinDir)JIT\jit64\mcc\**\*.pdb”>
```

Comment:
Oops, there are Unicode quotes - need to replace them with "plain" ones.

Reply:
Beautiful thanks